Lib/sqlite3/__main__.py

```diff
@@ -65,7 +65,10 @@ def runsource(self, source, filename="<input>", symbol="single"):
                 case "version":
                     print(f"{sqlite3.sqlite_version}")
                 case "help":
-                    print("Enter SQL code and press enter.")
+                    t = theme.syntax
+                    print(f"Enter SQL code or one of the below commands, and press enter.\n\n"
+                          f"{t.builtin}.version{t.reset}    Print underlying SQLite library version\n"
```

Comment:
Would it be worth mentioning `.help` as well for completeness? Something like 

```suggestion
                          f"{t.builtin}.version{t.reset}    Print underlying SQLite library version\n"
                          f"{t.builtin}.help{t.reset}       Show this help message\n"
```

Reply:
I considered it, but it seems unnecessary, it is recursive...

Reply:
FWIW, we do have precedent for this. CLIs created with `cmd` list the help command as part of the default help.

Reply:
```suggestion
                          f"{t.builtin}.version{t.reset}    Print underlying SQLite library version\n"
                          f"{t.builtin}.help{t.reset}       Print this help message\n"
```